src/sidebar/Header.tsx

```diff
@@ -28,21 +28,28 @@ import { isMV3 } from "@/mv3/api";
 import { DEFAULT_THEME } from "@/themes/themeTypes";
 
 const Header: React.FunctionComponent = () => {
-  const { logo, showSidebarLogo, customSidebarLogo } = useTheme();
-  const { baseThemeName } = useTheme();
+  const {
+    activeTheme: { logo, showSidebarLogo, customSidebarLogo, baseThemeName },
+    isLoading,
+  } = useTheme();
   /* In MV3, Chrome offers a native Close button */
   const showCloseButton = !isMV3();
 
+  const headerButtonClassName = cx({
+    [styles.button || ""]: true,
+    [styles.themeColorOverride || ""]: baseThemeName === DEFAULT_THEME,
+    [styles.themeColor || ""]: baseThemeName !== DEFAULT_THEME,
+  });
```

Comment:
I may have been confusing with my initial comment. No need for the `true` case:
```suggestion
  const headerButtonClassName = cx(
  styles.button,
  {
    [styles.themeColorOverride || ""]: baseThemeName === DEFAULT_THEME,
    [styles.themeColor || ""]: baseThemeName !== DEFAULT_THEME,
  });
```

Reply:
Gotcha, looks good to me